src/SFML/Graphics/Font.cpp

```diff
@@ -155,36 +153,32 @@ bool Font::loadFromFile(const std::filesystem::path& filename)
     // Cleanup the previous resources
     cleanup();
 
-    auto fontHandles = std::make_unique<FontHandles>();
+    auto fontHandles = std::make_shared<FontHandles>();
 
     // Initialize FreeType
     // Note: we initialize FreeType for every font instance in order to avoid having a single
     // global manager that would create a lot of issues regarding creation and destruction order.
-    FT_Library library;
-    if (FT_Init_FreeType(&library) != 0)
+    if (FT_Init_FreeType(&fontHandles->library) != 0)
     {
         err() << "Failed to load font (failed to initialize FreeType)\n" << formatDebugPathInfo(filename) << std::endl;
         return false;
     }
-    fontHandles->library.reset(library);
 
     // Load the new font face from the specified file
     FT_Face face;
-    if (FT_New_Face(library, filename.string().c_str(), 0, &face) != 0)
+    if (FT_New_Face(fontHandles->library, filename.string().c_str(), 0, &face) != 0)
```

Comment:
Is there a reason you are getting the font face in a temporary `FT_Face` instance which you later assign to `fontHandles->face` instead of getting it directly into `fontHandles->face` like you do with `fontHandles->library` and `fontHandles->stroker`?